.travis.yml

```diff
@@ -8,18 +8,19 @@ cache:
   directories:
   - node_modules
 rvm:
+  - 2.1
```

Comment:
Any reason to test against EOL rubies?
We *allow* usage with EOL rubies but not supporting it